server/http_integration_test.go

```diff
@@ -707,6 +707,46 @@ func TestHTTPServer_QueryEncoding(t *testing.T) {
 	shutdown()
 }
 
+func TestHTTPServer_TrailingSlash(t *testing.T) {
+	client := newClient()
+
+	config := "testdata/integration/endpoint_eval/11_couper.hcl"
+
+	type expectation struct {
+		Path string
+	}
+
+	shutdown, _ := newCouper(config, test.New(t))
+
+	t.Run("Query-Encoding", func(subT *testing.T) {
```

Comment:
Naming `Query-Encoding` does not match to the test content... a second case without trailing slash should be added.